### PR TITLE
docs: update information about db backend

### PR DESCRIPTION
**Pull request by robert-zaremba**
## Description

Adds instructions about recommended database.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -1,40 +1,36 @@
...
 
-build_tags = netgo
 
-#  experimental feature 
+#  experimental feature
 ifeq ($(EXPERIMENTAL),true)
 	build_tags += experimental
 endif
 
+build_tags = netgo
```
Comment (gsk967) [inline Makefile:33]: `build_tags = netgo` should be above the 
```#  experimental feature
ifeq ($(EXPERIMENTAL),true)
	build_tags += experimental
endif
```

Comment (robert-zaremba) [reply Makefile]: yes, thanks. updated

**Review by gsk967 — APPROVED** [review]
pre approving

File: README.md [hunk 2]
```diff
@@ -54,19 +54,40 @@ Building from source will automatically link the `libwasmvm.x86_64.so` created a
...
+
+Once you generate config files, you need to update:
+
+```bash
+# app.toml / base configuration options
+app-db-backend = "badgerdb"
+
+# config.toml / base configuration options
+db_backend = "badgerdb"
 ```
```
[the remark this answers is not in the document]
Comment (robert-zaremba) [reply README.md:91]: right, was testing something else

[the remark this answers is not in the document]
Comment (robert-zaremba) [reply README.md]: updated

**Pull request merged**